recipes/default.rb

```diff
@@ -122,20 +122,8 @@ class ::Chef::Resource
   backup false
 end
 
```

Comment:
Almost would have thought the default just includes the 3 other recipes. Did you want to leave this like this?